test/e2e/util.go

```diff
@@ -70,9 +70,9 @@ func waitForPodCondition(c *client.Client, ns, podName, desc string, condition p
 		if done {
 			return err
 		}
-		Logf("Waiting for pod %s status to be %q (found %q) (%d secs)", podName, api.PodRunning, pod.Status.Phase, time.Since(start).Seconds())
+		Logf("Waiting for pod %s status to be %q (found %q) (%.2f seconds)", podName, api.PodRunning, pod.Status.Phase, time.Since(start).Seconds())
 	}
-	return fmt.Errorf("gave up waiting for pod %s to be %s after %v", podName, desc, podStartTimeout)
+	return fmt.Errorf("gave up waiting for pod %s to be %s after %.2f seconds", podName, desc, podStartTimeout.Seconds())
```

Comment:
This way you are free to change to timeout constant and then something reasonable will happen. Happy to change it to `%.0f` if you prefer.